dev/date-picker.html

```diff
@@ -5,9 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Date picker</title>
-    <script type="module" src="./common.js"></script>
     <script type="module">
-      import '@vaadin/date-picker';
+      import '@vaadin/date-picker/theme/material/vaadin-date-picker.js';
```

Comment:
Is this change intentional?

Reply:
My bad, was testing some things. Reverted.